components/custom_open_pricing_modal_post_renderer/index.tsx

```diff
@@ -71,13 +74,22 @@ export default function OpenPricingModalPost(props: {post: Post}) {
 
     const openPricingModal = useOpenPricingModal();
 
+    const getUserIdsForUsersThatRequestedFeature = (requests: FeatureRequest[]): string[] => requests.map((request: FeatureRequest) => request.user_id);
     const postProps = props.post.props as Partial<CustomPostProps>;
     const requestFeatures = postProps?.requested_features;
     const wasTrialRequest = postProps?.trial;
+
+    useEffect(() => {
+        if (requestFeatures) {
+            for (const featureId of Object.keys(requestFeatures)) {
+                dispatch(getMissingProfilesByIds(getUserIdsForUsersThatRequestedFeature(requestFeatures[featureId])));
+            }
+        }
+    }, []);
```

Comment:
Do we need to add a condition to not always run this on every render?
```suggestion
    useEffect(() => {
        if (requestFeatures) {
            for (const featureId of Object.keys(requestFeatures)) {
                dispatch(getMissingProfilesByIds(getUserIdsForUsersThatRequestedFeature(requestFeatures[featureId])));
            }
        }
    }, [requestFeatures]);
```

Reply:
`[]` as second arg ensures it runs once

Reply:
Yes, you're right, we don't need to add anything else then